woocommerce/woocommerce-reset
# Add reset functionality for notes and settings

Add reset functionality for notes and settings

## Testing instructions

1. Clone this repo in your plugins folder and checkout this branch
2. Enable the WooCommerce Reset plugin
3. Open **WooCommerce > Home** and open your console
4. Run `wp.apiFetch({ method: 'DELETE', path: '/woocommerce-reset/v1/state'});`
5. Check the response to make sure all items come back with `true`
6. Refresh **WooCommerce > Home** it should re-trigger the onboarding process and the address field should be cleared.
